src/main/steps/index.ts

```diff
@@ -70,6 +80,19 @@ export const getNextIncompleteStepUrl = (req: AppRequest): string => {
       : 0;
   const url = getNextIncompleteStep(req.session.userCase, sequence[sequenceIndex], sequence, true);
 
+  const jurisdictionUrls = [
+    WHERE_YOUR_LIVES_ARE_BASED_URL,
+    JURISDICTION_DOMICILE,
+    JURISDICTION_LAST_TWELVE_MONTHS,
+    HABITUALLY_RESIDENT_ENGLAND_WALES,
+    LIVING_ENGLAND_WALES_SIX_MONTHS,
+    RESIDUAL_JURISDICTION,
+  ];
+
+  if (jurisdictionUrls.some(jurisdictionUrl => url.includes(jurisdictionUrl))) {
+    return `${CHECK_JURISDICTION}${queryString}`;
+  }
```

Comment:
@velimd yeah so if their nextIncompleteStep was one of the jurisdiction pages then we thought it would make sense to send them back to the start of the jurisdiction journey (CHECK_JURISDICTION) 
rather than a random page

What do you think?